src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/mapping/ClaimMapper.java

```diff
@@ -48,11 +48,11 @@ private ClaimData toClaimData(final String input) {
     }
 
     private ResponseData toNullableResponseData(final String input) {
-        return toNullableEntity(input, ResponseData.class);
```

Comment:
why toNullableEntity was wrong?

Reply:
@timja-kainos It looks like your code in git blame?

Reply:
reverted as it appearing after resolving conflicts.